### ISSUE TITLE
time difference

### ISSUE THREAD

**Issue by zzaebok** [zzaebok/AppUsage2Vec#1]
as-is: target app - previous app
to-be: latest used app - previous app

**Comment by zzaebok** [comment]
log 에 넣기 전에  relu (max) 씌우기

**Comment by zzaebok** [comment]
remove utils

**Comment by zzaebok** [comment]
preds = torch.topk(scores, dim=1, k=self.k).indices  # [batch_size, k]
            indicator = torch.sum(torch.eq(preds, targets), dim=1)  # [batch_size]
            coefficient = torch.pow(torch.Tensor([self.alpha] * indicator.size(0)).to(self.device), indicator) # [batch_size]
            
            loss = F.cross_entropy(scores, targets.view(-1), reduction='none')
            loss = torch.mean(torch.mul(coefficient, loss))
